documentation/app/templates/components/list-item-template.hbs

```diff
@@ -0,0 +1,849 @@
+              (container)</Hds::Link::Inline>. It can also be nested within
+            other components as a template.</li>
+        </ul>
+        <img
+          src="/images/list-item-template-usage.svg"
+          alt="List Item demo with List Item Template being used inside of it"
+        />
+
+        <h3 id="when-not-to-use">When not to use</h3>
+        <ul>
+          <li>When your content within is more than 3 lines tall, consider
+            creating your own template.</li>
+          <li>If you need to make use of styles or nested components that are
+            not already available within the template.</li>
+        </ul>
+
+        <h2 id="interactivity-and-actions">Interactivity & actions</h2>
+        <p>By default the List Item Template is static in nature. Interactivity
+          can be applied to elements within, but should adhere to the following
+          guidelines.</p>
+
+        <h3 id="tooltips">Tooltips</h3>
+        <p>Tooltips can be paired with the template for text, icon, or badge
+          elements that need to provide more context on hover. For example, when
+          the List Item Template is within a list item (container).</p>
+        <img
+          src="/images/list-item-template-tooltips.svg"
+          alt="Demonstration of tooltips on the List Item Template"
+        />
+
+        <h3 id="interactivity-specific-elements">Interactivity on a specific
+          element</h3>
+        <ul>
+          <li>
+            If paired with the List Item (container) the state of the container
+            should be set to “static”, meaning that clicking on the entire list
+            is non-interactive and only individual elements would be.
+          </li>
+          <li>
+            On hover of clickable data, the background color of the object
+            hovered should change to “Surface/interactive-hover”. If the action
+            is to copy text then the default copy icon will show.
+          </li>
+          <li>
+            When adding interactivity to an individual element such as a piece
+            of text, consider adding a link style (underline or color change) to
+            signify that it is interactive if this is the main action of the
+            list item.
+          </li>
+        </ul>
+
+        <img
+          src="/images/list-item-template-interactivity.svg"
+          alt="Demonstration of List Item Template interactivity"
+        />
+
+        <h2 id="color">Color</h2>
+        <h3 id="transparent-background">Transparent background</h3>
+        <p>The List Item Template has a transparent background color by default.
+          It is set to transparent so it utilizes the background color of
+          whatever component it is nested within. This can be changed manually
+          as needed.</p>
+        <h3 id="title">Title</h3>
+        <p>Any text used within the “title” should use HDS text style of
+          “Foreground/Strong”.</p>
```

Comment:
Similar to prev, wonder if it would be helpful to link to the HDS tokens https://helios.hashicorp.design/foundations/colors?tab=palette#foreground-1. 

Are there guidelines around the font style and weight in addition to color, or are these applied automatically? https://helios.hashicorp.design/foundations/typography?tab=code#style-and-weight